common/djangoapps/track/tests/test_shim.py

```diff
@@ -149,3 +155,167 @@ def test_no_course_id(self):
             'timestamp': FROZEN_TIME,
         }
         assert_events_equal(expected_event, emitted_event)
+
+
+@override_settings(
+    EVENT_TRACKING_BACKENDS={
+        '0': {
+            'ENGINE': 'eventtracking.backends.routing.RoutingBackend',
+            'OPTIONS': {
+                'backends': {
+                    'first': {'ENGINE': 'track.tests.InMemoryBackend'}
+                },
+                'processors': [
+                    {
+                        'ENGINE': 'track.shim.GoogleAnalyticsProcessor'
+                    }
+                ]
+            }
+        },
+        '1': {
+            'ENGINE': 'eventtracking.backends.routing.RoutingBackend',
+            'OPTIONS': {
+                'backends': {
+                    'second': {
+                        'ENGINE': 'track.tests.InMemoryBackend'
+                    }
+                }
+            }
+        }
+    }
+)
+class MultipleShimGoogleAnalyticsProcessorTestCase(EventTrackingTestCase):
```

Comment:
No, sorry about that.. I only needed part of the commit, and have reverted the rest with [b8cdf17](https://github.com/hks-epod/edx-platform/pull/1/commits/b8cdf17e4e24b9bac1fa0326a49e73bd41dea108)